### PR TITLE
Issue #3062716 by mdeny, agami4: Collapse email notification digest settings by default

### PR DESCRIPTION
**Pull request by denysyukm**
## Problem
As a LU I want to have the notification settings in a clearer overview

## Solution
Rework output of email notification settings

## Issue tracker
https://jira.goalgorilla.com/browse/FOR-248
https://www.drupal.org/project/social/issues/3062716

## How to test
- [ ] Go to user edit page
- [ ] You should see email notifications section as collapsed fieldset
- [ ] You should see email notifications items as collapsed fieldsets

## Release notes
Collapse email notification digest settings by default

### CODE REVIEW

File: modules/custom/activity_send/modules/activity_send_email/css/activity_send_email.admin.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+      @media(min-width: 1200px) {
+        flex-basis: 70%;
+      }
+
+    }
+
+  }
+
+}
```
Comment (OH17L) [inline modules/custom/activity_send/modules/activity_send_email/css/activity_send_email.admin.scss]: Add the empty line.

File: modules/custom/activity_send/modules/activity_send_email/js/collapse.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+(function ($, Drupal) {
+
+  Drupal.behaviors.socialActivitySendEmail = {
+    attach: function attach(context) {
+      var $context = $(context);
+
+      $context.find('.form-email-notification .card__title > a').each(function () {
+        $(this).on('click', function () {
+          var $icon = $(this).find(' svg');
```
Comment (OH17L) [inline modules/custom/activity_send/modules/activity_send_email/js/collapse.js]:
```suggestion
          var $icon = $(this).find('.icon');
```

File: modules/custom/activity_send/modules/activity_send_email/js/collapse.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+          if ($icon.hasClass('icon-expand_more')) {
+            $icon.html('<use xlink:href="#icon-expand-less" />');
+          }
+          else {
+            $icon.html('<use xlink:href="#icon-expand_more" />');
+          }
+
+          $icon.toggleClass('icon-expand_more', 'icon-expand-less');
```
Comment (OH17L) [inline modules/custom/activity_send/modules/activity_send_email/js/collapse.js]: The second argument of the method should be a boolean value.

File: modules/custom/activity_send/modules/activity_send_email/activity_send_email.libraries.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+admin:
```
Comment (OH17L) [inline modules/custom/activity_send/modules/activity_send_email/activity_send_email.libraries.yml]: Add dependencies.

File: modules/custom/activity_send/modules/activity_send_email/js/collapse.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  Drupal.behaviors.socialActivitySendEmail = {
+    attach: function attach(context) {
+      var $context = $(context);
+
+      $context.find('.form-email-notification .card__title > a').each(function () {
+        $(this).on('click', function () {
+          var $icon = $(this).find(' svg');
+
+          if ($icon.hasClass('icon-expand_more')) {
+            $icon.html('<use xlink:href="#icon-expand-less" />');
```
Comment (OH17L) [inline modules/custom/activity_send/modules/activity_send_email/js/collapse.js]: I am not sure but maybe will be better to change just the **xlink:href** attribute value.

**Comment by agami4** [comment]
@OH17L,
I've updated collapse email notification settings, please check

**Comment by jaapjan** [comment]
Behat test is still failing: https://travis-ci.org/goalgorilla/open_social/jobs/557357643 Please fix, remember there is a feature stop coming next week so please do so in time so we still have time to properly review..

**Comment by denysyukm** [comment]
@jaapjan, I fixed behat test. Could you check travis-ci? I found there unclear issue: https://travis-ci.org/goalgorilla/open_social/jobs/589304156

**Comment by ronaldtebrake** [comment]
Merged in latest changes to see if that helps your travis PR. Also base branch was on 6.x, maybe that was what caused travis to fail before.

**Comment by ronaldtebrake** [comment]
This needs a rebase, you can open up a new PR on 7.x so travis will be fired.

**Comment by denysyukm** [comment]
@ronaldtebrake, for some reason I don't see travis check here. Could you take a look and let me know what happens?

**Comment by ronaldtebrake** [comment]
> @ronaldtebrake, for some reason I don't see travis check here. Could you take a look and let me know what happens?

@denysyukm That's because it needs a rebase. You need to create a new PR based on the 8.x-7.x branch. Since it was branched of the old 8.x-6.x switching it will break travis.

You can just cherry-pick your commits in to a new PR and open it there and reference that one to this PR.

**Comment by denysyukm** [comment]
@ronaldtebrake, I created new PR based on the 8.x-7.x: [1517](https://github.com/goalgorilla/open_social/pull/1517)

**Comment by ronaldtebrake** [comment]
Thanks will close this one and merge the other when green :D

**Pull request closed without merging**